snapcraft.yaml

```diff
@@ -915,13 +931,22 @@ parts:
       - lib/*/libnuma*so*
       - lib/*/libpixman*so*
       - lib/*/libusbredir*so*
+      - lib/*/libfdt*.so*
       - share/qemu/keymaps*
-      - share/qemu/efi-virtio.rom*
-      - share/qemu/kvmvapic.bin*
-      - share/qemu/s390-*.img*
-      - share/qemu/slof.bin*
-      - share/qemu/vgabios-*.bin*
       - share/qemu/bios-256k.bin*
```

Comment:
There is only one `bios-256k.bin` file in the seabios package so I'd drop the `*`.

Reply:
this is to make them option on the non-qemu archs.

Reply:
Ah, I thought that for that you needed a `-` prefix like `-bin/vgimportclone` we have elsewhere.

Reply:
maybe thats another way! does that mean optional?

Reply:
Yes, `-bin/foo` means it will tentatively copy it but won't complain if it cannot.

Reply:
I couldn't find any official reference though but tests confirmed that `-` makes the copy optional.